api/frontend.proto

```diff
@@ -186,15 +186,19 @@ service FrontendService {
     };
   }
 
+  // GetBackfill returns a backfill object by its ID.
+  //
   // BETA FEATURE WARNING:  This call and the associated Request and Response
   // messages are not finalized and still subject to possible change or removal.
   rpc GetBackfill(GetBackfillRequest) returns (Backfill) {
     option (google.api.http) = {
       get: "/v1/frontendservice/backfills/{backfill_id}"
     };
   }
-
-  // Side effects: Any tickets waiting for this backfill will be returned to the active pool, no longer pending.
+  
+  // UpdateBackfill updates a Backfill object with an ID set and SearchFields defined.
```

Comment:
extensions will be updated to.  Perhaps something like "UpdateBackfill updates search_fields and extensions for the backfill with the provided id"?

Reply:
Applied!